### PR TITLE
Documentation on GitHub Flow and Deployment practices for Mozilla Fou…

### PR DESCRIPTION
**Pull request by cadecairos**
…ndation software projects

@MozillaFoundation/production R?

### CODE REVIEW

**Comment by alanmoo** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ We enforce our coding standards with automated tooling ("linters"), that are usu
 
 Refer to [Mofo Style](https://github.com/MozillaFoundation/mofo-style) for more info.
 
-## Git Flow
+## GitHub Flow
 
-Many of our repos follow the Git Flow convention. If the default branch is `develop`, it is likely that this is the case.
+Active projects and future projects will use GitHub flow for branching. This means that new features should be written in feature branches based on `master`. When a feature is **complete**, open a pull request against `master` to have the new feature (or fix) brought into the base branch.
```
Comment (Pomax) [inline README.md]: we don't want to give up early review, though - catching methodology problems early saves a lot of time fixing later, so it's a good idea to mention that we're not "forbidding" Work In Progress PRs.

Comment (cadecairos) [inline README.md]: You're right, I'll try to clarify what I was trying to say here.

File: README.md [hunk 2]
```diff
@@ -34,11 +34,13 @@ We enforce our coding standards with automated tooling ("linters"), that are usu
...
 
-Many of our repos follow the Git Flow convention. If the default branch is `develop`, it is likely that this is the case.
+Active projects and future projects will use GitHub flow for branching. This means that new features should be written in feature branches based on `master`. When ready, open a pull request against `master` to have the new feature (or fix) brought into the base branch. Branches should only merge into master when fully completed, so other people's work is never blocked.
 
-For more details, see: [http://nvie.com/posts/a-successful-git-branching-model](http://nvie.com/posts/a-successful-git-branching-model)
+For more details, see: [https://guides.github.com/introduction/flow/](https://guides.github.com/introduction/flow/)
+
+**NOTE**: The decision to standardize GitHub flow on Mozilla Foundation projects was made In August 2016. Therefore, you can expect to find some projects that have yet to be updated to use GitHub flow. You will be able to identify these projects by checking what the base branch of the repository it. If it is `develop` you should use [Git Flow](http://nvie.com/posts/a-successful-git-branching-model), otherwise, use GitHub flow.
```
Comment (gideonthomas) [inline README.md:43]: lowercase i in In :)

File: README.md (hunk 1 above)
Comment (cadecairos) [inline README.md]: @Pomax Fixed up.

**Pull request merged**